### ISSUE TITLE
Training for a new dataset.

### ISSUE THREAD

**Issue by KamalAwasthi** [co60ca/EmotionNet2#6]
During the face extraction, you are mapping labels to each file. But where are saving those labels? 

Actually, I want to run EmotionNet2 for my own dataset. I want to finetune your model. As far as I can see I would have to change the code so as to train the model as per the labels for my dataset. Unlike KDEF, my dataset does not have labels embedded into the filename. I have a separate csv file for it. 

Can you please clear this up?

**Comment by co60ca** [comment]
Hi,

I'm not 100% sure what you are saying. But the loader in pytorch gets the label from the folder name. If you want to do your own labels you can change the data loader or replicate the way I load the way by inserting each class into its respective folder. You should see that training will have different folders for each class.

See loader on this line:
https://github.com/co60ca/EmotionNet2/blob/master/python/emotionnet.py#L172

Please close if I've answered your question. If you need more assistance I can clarify.

**Comment by KamalAwasthi** [comment]
Yeah! I got your point. I have four classes and what I need to do is to just make four folders each with the name of a class. Then the loader will get its label from those folder names. I think this is what you are trying to infer here,right?

**Comment by KamalAwasthi** [comment]
And one more thing, If I do make these folders and put my images into them. Do names of images inside those folders matter? I mean It does matter for KDEF while you are extracting faces but once you are done with face extraction and putting them into respective folders, those images can take any name?

**Comment by co60ca** [comment]
Yes Kamal, this should be a problem with regard to the names. The loader is from pytorch so you can refer to that documentation.

**Comment by KamalAwasthi** [comment]
Thank You very much!

I think you have cleared my all queries. Closing now.
